crates/tools/src/main.rs

```diff
@@ -171,6 +173,49 @@ fn main() {
                         .required(true)
                         .help("The output scripts deploy json file path"),
                 ),
+        )
+        .subcommand(
+            SubCommand::with_name("prepare-pk")
+                .about("Generate godwoken nodes private keys, poa and rollup configs")
+                .arg(arg_privkey_path.clone())
+                .arg(
+                    Arg::with_name("ckb-count")
```

Comment:
```suggestion
                    Arg::with_name("capacity")
```

Please don't reinvention term.